addon/components/power-calendar/days.js

```diff
@@ -199,7 +209,7 @@ export default Component.extend({
     }
 
     let disabledDates = this.get('disabledDates');
-    if (disabledDates && disabledDates.some((d) => momentDate.isSame(d, 'day'))) {
+    if (disabledDates && disabledDates.some((d) => momentDate.isSame(d, 'day') || (WEEK_DAYS.includes(d) && momentDate.format('ddd') === d))) {
```

Comment:
There is some flaw in the logic there, as `WEEK_DAYS.includes(d)` needs to be part of the callback of the `some()` function.